src/app/screens/LNURLPay/index.tsx

```diff
@@ -318,8 +297,16 @@ function LNURLPay() {
   return (
     <>
       <div className="flex flex-col grow overflow-hidden">
-        <ScreenHeader title={tCommon("actions.send")} />
-        {!successAction ? (
+        <ScreenHeader
+          title={
+            !result
```

Comment:
Is this a ternary if/else-if? Might be easier to read to move this into a function?